src/main/controllers/helpers/CaseRoleHelper.ts

```diff
@@ -0,0 +1,9 @@
+import { AppRequest } from '../../definitions/appRequest';
+import { getCaseApi } from '../../services/CaseService';
+
+import { getLanguageParam } from './RouterHelpers';
+
+export const removeClaimantRepresentative = async (req: AppRequest): Promise<string> => {
+  await getCaseApi(req.session.user?.accessToken)?.removeClaimantRepresentative(req);
+  return '/citizen-hub/' + req.session.userCase.id + '?language=' + getLanguageParam(req.url);
```

Comment:
getLanguageParam would return '?lng=en' or '?lng=cy'
Is there any special reason to add '?language=' here?